os/arch/arm/src/amebad/amebad_i2s.c

```diff
@@ -0,0 +1,1818 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+
+#include <sys/types.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <semaphore.h>
+#include <errno.h>
+#include <assert.h>
+#include <queue.h>
+#include <debug.h>
+
+#include <tinyara/irq.h>
+#include <tinyara/arch.h>
+#include <tinyara/kmalloc.h>
+#include <tinyara/wdog.h>
+#include <tinyara/wqueue.h>
+#include <tinyara/audio/audio.h>
+#include <tinyara/audio/i2s.h>
+#include <tinyara/mm/mm.h>
+
+#include "irq/irq.h"
+
+#include <arch/board/board.h>
+#include "amebad_i2s.h"
+
+#include "mbed/targets/hal/rtl8721d/PinNames.h"
+#include "mbed/hal_ext/i2s_api.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+//#ifndef CONFIG_SCHED_WORKQUEUE
+//#error Work queue support is required (CONFIG_SCHED_WORKQUEUE)
+//#endif
+
+#ifdef CONFIG_AUDIO
```

Comment:
@jeongchanKim , Do I need to remove the macro `CONFIG_AUDIO` added here?

> Please modify the comment, CONFIG_AUDIO.
The comment at the end of this file was added according to this macro.

Reply:
Sorry, it is okay.